BridgeAppSDK/Resources/SBAResourceFinder.swift

```diff
@@ -82,12 +82,41 @@ public class SBAResourceFinder: NSObject {
     }
     
     public func htmlNamed(resourceNamed: String) -> String? {
-        if let data = self.dataNamed(resourceNamed, ofType: "html") {
-            return String(data: data, encoding: NSUTF8StringEncoding)
+        if let data = self.dataNamed(resourceNamed, ofType: "html"),
+            let html = String(data: data, encoding: NSUTF8StringEncoding) {
+            return importHTML(html)
         }
         return nil
     }
     
+    func importHTML(input: String) -> String {
+        
+        // setup string
+        var html = input
+        func search(str: String, _ range: Range<String.Index>?) -> Range<String.Index>? {
+            return html.rangeOfString(str, options: .CaseInsensitiveSearch, range: range, locale: nil)
+        }
+        
+        // search for <import href="resourceName.html" /> and replace with contents of resource file
```

Comment:
Because this is designed for override, the replacement html may or may *not* be in the same bundle as the original html. This was a way that I could replace the contact and IRB text with the appropriate text for Lux without have to include a full translation for all the html files that use it.